main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"net/http"
+	_ "net/http/pprof"
+
+	"github.com/segmentio/rate-limiting-log-proxy/container"
+	"github.com/segmentio/rate-limiting-log-proxy/logger"
+	syslog "gopkg.in/mcuadros/go-syslog.v2"
+)
+
+func main() {
+	// Start http server for pprof
+	go func() {
+		log.Println(http.ListenAndServe("localhost:6060", nil))
+	}()
+
+	// Start syslog server
+	server := syslog.NewServer()
+	server.SetFormat(syslog.RFC3164)
+	server.ListenUDP("0.0.0.0:10514")
```

Comment:
Yeah, as per the readme, I still need to move this config out into flags.  Things that should be configurable:
- docker host
- rate limit settings
- pprof port (or disable)
- server port/proto